lib/rubocop/cop/performance/size.rb

```diff
@@ -0,0 +1,44 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop is used to identify usages of `count` on an
+      # `Array` and `Hash` and change them to `size`.
+      #
+      # @example
+      #   # bad
+      #   [1, 2, 3].count
+      #
+      #   # bad
+      #   {a: 1, b: 2, c: 3}.count
+      #
+      #   # good
+      #   [1, 2, 3].size
+      #
+      #   # good
+      #   {a: 1, b: 2, c: 3}.size
+      class Size < Cop
+        MSG = 'Use `size` instead of `count`.'
+
+        def on_send(node)
+          receiver, method = *node
+          block, = *node.ancestors
```

Comment:
`block = node.parent` is the same. I would put it further down, just before it's used, or perhaps just put
```ruby
return if node.parent && node.parent.block_type?
```
on line 31.